lib/tasks/tests-task.ts

```diff
@@ -0,0 +1,21 @@
+import { ITask, IProject, ICheckupResult, ITaskItemData } from '../../interfaces';
+import FileSearcherTask from '../file-searcher-task';
+
+const SEARCH_PATTERNS = {
+  unit: ['**/tests/unit/**/*.js'],
+  acceptance: ['**/tests/acceptance/**/*.js'],
+  integration: ['**/tests/integration/**/*.js'],
+};
+
+export default class TestsTask extends FileSearcherTask implements ITask {
+  constructor(project: IProject, result: ICheckupResult) {
+    super(project, result, SEARCH_PATTERNS);
+  }
+
+  async run() {
+    let result: ITaskItemData = await this.searcher.search();
```

Comment:
You can probably just assign this directly to `this.result.tests` and skip creating an intermediate var.